locales/src/en/translation.json

```diff
@@ -214,7 +214,7 @@
     },
     "error": {
         "code": "Error Code: __code__",
-        "dataloadfailed": "Failed to load data from __url__",
+        "dataloadfailed": "Failed to load __filename__.",
```

Comment:
I normally think we should try to not change existing strings since they'll remove a translation that we may never get back. However, I wanted to use something that we could also use in enketo-core and think the filename is nicer than the URL (as we have in enketo-core). The old string would likely not be used anymore. I'll create a PR to use the same string in Enketo Core once agreed.

Reply:
Right now core uses the unlocalized "Can't find" prefix, right? https://github.com/enketo/enketo-core/blob/c5005f7733147b444c5adbf4fa8014a27cb250de/src/js/form-model.js#L214

I'm happy with either "Failed to load" or "Can't find". Agreed it would be nice to localize and that just referencing the expected filename is much better than using the URL.

Reply:
> Right now core uses the unlocalized "Can't find" prefix, right?

Yes, indeed.